docker/base/dockerfile.merlin

```diff
@@ -82,27 +82,16 @@ RUN apt update -y --fix-missing && \
 
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
-# Install multiple packages
+# Install Python packages
 
 # cmake 3.25.0 broke find_package(CUDAToolkit), which breaks the FAISS build:
 # https://gitlab.kitware.com/cmake/cmake/-/issues/24119
 # A fix has already been merged but not yet released:
 # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/7859
-RUN pip install "cmake<3.25.0" ninja scikit-build
-
-RUN pip install pandas==1.3.5
-RUN pip install cupy-cuda117 nvidia-pyindex pybind11 pytest protobuf transformers==4.12 tensorflow-metadata
-RUN pip install betterproto cachetools graphviz nvtx scipy sklearn
-RUN pip install tritonclient[all] grpcio-channelz fiddle wandb npy-append-array
-RUN pip install git+https://github.com/rapidsai/asvdb.git@main
-# xgboost and treelite are pinned from tritonserver support
-RUN pip install xgboost==1.6.2 lightgbm treelite==2.3.0 treelite_runtime==2.3.0
-RUN pip install lightfm implicit
-RUN pip install numba
-RUN pip install "cuda-python>=11.5,<12.0"
-RUN pip install fsspec==2022.5.0 llvmlite
-RUN pip install pynvml dask==${DASK_VER} distributed==${DASK_VER}
```

Comment:
do we intend to remove the pinned dask version in the requirement file?

Reply:
They're actually [installed twice](https://github.com/NVIDIA-Merlin/Merlin/pull/741/files#diff-0b49c155ea357cdfbade44be15597ce3a854bb3f829c69de306108c2f3db5ad9R112), so it seems safe to remove them here? @jperez999, is there a reason to install these libraries here and then again later?

Reply:
It's not safe because packages that are installed in between change the versions